test/helpers/test_provisioner.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require_relative './kubeclient_helper'
+
+class TestProvisioner
+  extend KubeclientHelper
+
+  class << self
+    def prepare_cluster
+      WebMock.allow_net_connect!
+      $stderr.print "Preparing test cluster... "
```

Comment:
Why stderr?

Reply:
STDERR is the normal descriptor for diagnostic messages. Kir and Simon taught me about this when we started the gem: https://www.jstorimer.com/blogs/workingwithcode/7766119-when-to-use-stderr-instead-of-stdout. Most of this gem's logging has been done to STDERR since https://github.com/Shopify/kubernetes-deploy/pull/5/files#r97137080; our "result" is pretty much just the exit code.